src/utils.js

```diff
@@ -79,12 +80,19 @@ const fileNameFlag = Flags.string({
 
 const startTimeFlag = Flags.string({
 	description: 'Start time for the logs in UTC',
-	required: true,
 });
 
 const endTimeFlag = Flags.string({
 	description: 'End time for the logs in UTC',
-	required: true,
+});
+
+const pastFlag = Flags.string({
+	description: 'Past time window in mins',
+});
+
+const fromFlag = Flags.string({
+	description:
+		'From time in DD-MM-YYYY:HH:MM:SS format to fetch logs from the past. Used as the starting time for the past time duration.',
```

Comment:
@AjazSumaiya can we add a note here that, from flag takes system time?

Reply:
summary: 'brief summary',                    // help summary for flag
You can add extra details here